### ISSUE TITLE
Diff for local_file skips last line

### ISSUE THREAD

**Issue by kenske** [dmlittle/scenery#18]
If I use an aws `local_file` resource, the last line of `content` is not included in the diff output. As a workaround I added a newline to the end of my templates.

**Comment by dmlittle** [comment]
Hi @kenske, thanks for opening up an issue. Can you provide a sample terraform file I can use to reproduce the issue? It sounds like you've found a workaround but ideally you shouldn't have to!

**Comment by kenske** [comment]
Here is an example:

```terraform
# main.tf
data "template_file" "config" {
  template = "${file("template.yml")}"
}

resource "local_file" "config" {
  content  = "${data.template_file.config.rendered}"
  filename = "output.yaml"
}
```

```yaml
# template.yaml
this:
  is:
    an:
      -example
```

If I don't add a newline at the end of `template.yaml`, this is the output I get from scenery:

```bash
$ terraform plan | scenery
+ local_file.config
    id:       <computed>
    content:  "this:
                 is:
                   an:"
    filename: "output.yaml"
```

**Comment by kenske** [comment]
I created #19 to fix the issue, but I'm not sure if it's the best solution.

**Comment by dmlittle** [comment]
@kenske thanks for the report. I'm able to reproduce the bug when the template file doesn't have a trailing newline at the end of the file. While it is good practice to have one, scenery shouldn't be truncating the output if it doesn't. 

```bash
# without trailing new line (% at the end of example represents there is no new line)
> cat template.yml
this:
  is:
    an:
      -example%                                                                                                                                                                                                                               

# with trailing new line
> cat template.yml
this:
  is:
    an:
      -example
```

The approach you have in #19 is the write approach. I prematurely added that formatting truncation that introduced this bug. I left some comments in your PR and it needs to be rebased before I can merge it but thanks for helping improve scenery! 

Let me know if there is anything I can do to help you get the PR through. If you feel you don't have time to work on this issue let me know and I can take over it.

**Comment by dmlittle** [comment]
Closed in #19
